src/hooks/useBulkActions.ts

```diff
@@ -31,3 +31,14 @@ function useBulkActions(initial: string[] = []) {
   };
 }
 export default useBulkActions;
+
+export interface BulkActions {
+  add: (data: string) => void;
+  isSelected: (data: string) => boolean;
+  listElements: string[];
+  remove: (data: string) => void;
+  reset: () => void;
+  set: (data: string[]) => void;
+  toggle: (data: string) => void;
+  toggleAll: (itmes: Node[], selected: number) => void;
+}
```

Comment:
I don't see this interface is used at all.

Reply:
wip leftover, removing